tests/admin_inlines/test_inline_collapse.py

```diff
@@ -0,0 +1,76 @@
+"""
```

Comment:
Thank you for writing tests, unfortunately none of them test the bug fix around the width calculation issue